aws_fuzzy_finder/main.py

```diff
@@ -126,7 +139,11 @@ def choice(fuzzysearch_bash_command):
     except subprocess.CalledProcessError:
         exit(1)
 
-    return choice.split(SEPARATOR)[1].rstrip()
+    if use_ssm:
+        output = choice.split(' ')[1].replace('(', '').replace(')', '').rstrip()
```

Comment:
instead of stripping out the IP address, I have it pulling the instance id from the string.